config/samples/conduit-with-schema-registry.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: operator.conduit.io/v1alpha
+kind: Conduit
+metadata:
+  name: conduit-generator-schema-registry
+spec:
+  running: true
+  name: generator.standalone.log
+  description: generator pipeline
+  schemaRegistry:
+    url: http://schema-registry-apicurio-registry.tenant:8080/apis/ccompat/v7
+  connectors:
+    - name: source-connector
+      type: source
+      plugin: conduitio/conduit-connector-generator
+      settings:
+        - name: format.type
+          value: structured
+        - name: format.options.id
+          value: "int"
+        - name: format.options.name
+          value: "string"
+        - name: format.options.company
+          value: "string"
+        - name: format.options.trial
+          value: "bool"
+        - name: recordCount
+          value: "3"
+    - name: destination-connector
+      type: destination
+      plugin: conduitio/conduit-connector-log
```

Comment:
this is unrelated to this PR, but I wonder if this should be structured such as:

```yaml
plugin:
  name:
  version:
```

or 

```yaml
pluginName: ...
pluginVersion: ...
```

Reply:
There is actually quite a few of them, as per:

https://github.com/ConduitIO/conduit-operator/blob/e5f2ee19c4f01bc8cf57b0a3d3e3054e8d1fb16d/api/v1alpha/conduit_types.go#L78-L81

Ideally this could have been combined in the same field. I can file this as a later thing, but it is a bit of a change.
Mostly converting a field from one type to another (string to object) will need to be done with some work.

I prefer to punt on this until later on.

Reply:
@raulb filed issue here https://github.com/ConduitIO/conduit-operator/issues/42